tools/isolationchamber.c

```diff
@@ -37,10 +39,13 @@ void extra_create() {
 }
 
 void enter_signal( object item, object oldenv, object mover, int pass ) {
-     if ( objectp( item ) && is_wizard( item ) )
+     if ( objectp( item ) )
```

Comment:
Removing the is_wizard() check is a good call if mortals are also protected from being slooped, which they should be.